nette/notejam/app/presenters/templates/@layout.latte

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<!--[if lt IE 7 ]>
+<html class="ie ie6" lang="en"> <![endif]-->
+<!--[if IE 7 ]>
+<html class="ie ie7" lang="en"> <![endif]-->
+<!--[if IE 8 ]>
+<html class="ie ie8" lang="en"> <![endif]-->
+<!--[if (gte IE 9)|!(IE)]><!-->
+<html lang="en"> <!--<![endif]-->
+<head>
+
+	<!-- Basic Page Needs ================================================== -->
+	<meta charset="utf-8">
+	<title>Notejam{ifset title}: {include title|striptags}{/ifset}</title>
+	<meta name="description" content="">
+	<meta name="author" content="">
+
+	<!-- Mobile Specific Metas ================================================== -->
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+
+	<!-- CSS ================================================== -->
+	<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/skeleton/1.2/base.min.css">
+	<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/skeleton/1.2/skeleton.min.css">
+	<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/skeleton/1.2/layout.css">
+	<link rel="stylesheet" href="{$basePath}/css/style.css">
+
+	<!--[if lt IE 9]>
+	<script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+	<![endif]-->
+</head>
+<body>
+<div class="container">
+	{if $user->isLoggedIn()}
+		{control userBar}
+	{else}
+		<div class="sixteen columns">
+			<div class="sign-in-out-block">
+				<a n:href="Sign:up">Sign up</a>&nbsp;&nbsp;&nbsp;<a n:href="Sign:in">Sign in</a>
+			</div>
+		</div>
+	{/if}
+
+	<div class="sixteen columns">
+		<h1 class="bold-header"><a n:href="Homepage:" class="header">note<span class="jam">jam: </span></a>
+			<span>{include header}</span>
+		</h1>
+	</div>
+
+	{ifCurrent Account:settings}
+	{else}
```

Comment:
this is not very nice usage... it might be a bit longer, but `{if $presenter->isLinkCurrent('Account:settings')}`is more straightforward